govc/test/library.bats

```diff
@@ -50,7 +50,14 @@ load test_helper
   run govc tags.attached.ls "$tag_name"
   assert_success ""
 
-  run govc library.rm /my-content
+  run govc library.update -n new-content my-content
+  assert_success
+
+  run govc library.info new-content
+  assert_success
+  assert_matches "$id"
+
+  run govc library.rm /new-content
```

Comment:
interesting: `rm` needs `/` whereas the other commands don't?

Reply:
The library.* commands all use the same library/finder package, the first `/` is optional in all cases.